BCC.md

```diff
@@ -25,6 +25,8 @@ run the following command pointing adeb to the kernel sources which will
 have it extract headers from there and push them to the device.
 ```
 adeb prepare --build --bcc --kernelsrc /path/to/kernel-source/
+OR(If you build kernel out-of-tree):
+adeb prepare --build --bcc --kbuildout /path/to/kernel-kbuild_out-dir/
```

Comment:
Don't you need to point to both the in-tree and out-of-tree directories in this command?

Reply:
No.
In my script, kbuildbout parameter is ignored after giving kernesrc parameter.
--kernelsrc will be used in the cases that kernel in built in-tree.
--kbuildout will used in the cases that kernel is build out-of-tree. When building kernel out-of-tree, "source" directory is linked to the original kernel source directory. The script will deal with this case.